packages/node_modules/@ciscospark/react-component-utils/src/activity.js

```diff
@@ -5,7 +5,7 @@ import {isImage} from './files';
  * Constructs a default activity
  *
  * @export
- * @param {any} conversation
+ * @param {object} conversation
```

Comment:
I don't think conversation is guaranteed to be a object right? I think this is set up to also take conversation as just a conversation id.